src/sparsezoo/model/model.py

```diff
@@ -156,6 +159,11 @@ def __init__(self, source: str, download_path: Optional[str] = None):
             files, display_name="benchmarks.yaml"
         )
         self.eval_results: File = self._file_from_files(files, display_name="eval.yaml")
+        # plaintext validation metrics optionally parsed from a zoo stub
+        self.validation_results: Optional[List[ModelResult]] = validation_results
```

Comment:
Why not `Optional[Dict[ModelResult]]` to enable easier access to the results (by key and not by index)?

Reply:
nice added!